tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -2,6 +2,14 @@
 
 from coalib.bearlib.aspects.collections import aspectlist
 from coalib.bearlib.aspects.Metadata import Metadata
+from coalib.bears.LocalBear import LocalBear
```

Comment:
IMO it is better to better to define this bear in `conftest.py` because we may further need this bear and its settings for future tests.